packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -1,29 +1,53 @@
-import { Button, Box, Grid } from 'grommet'
-import { FormNext, FormPrevious } from 'grommet-icons'
+import { Button, Box, Grid, Select, ThemeContext } from 'grommet'
+import {
+  CirclePlay,
+  Pause,
+  FormNext,
+  FormPrevious,
+  FormDown
+} from 'grommet-icons'
 import PropTypes from 'prop-types'
-import React from 'react'
+import React, { useEffect, useRef, useState } from 'react'
 import styled, { withTheme } from 'styled-components'
 import { useTranslation } from 'react-i18next'
 
+import controlsTheme from './theme'
 import locationValidator from '../../../helpers/locationValidator'
 
-const StyledButton = styled(Button)`
+const DirectionButton = styled(Button)`
   & > div {
     flex-direction: column;
   }
 `
 
+const ThumbnailButton = styled(Button)`
+  display: flex;
+  height: 40px;
+  width: 40px;
+  padding: 0;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
```suggestion
  background-position: center;

  &[aria-selected=true] {
    border: solid 2px ${props => props.theme.global.colors['neutral-2']};
  }
```

You can use an attribute selector to add a border to the active button.